README.md

```diff
@@ -13,43 +13,43 @@ Our motto:
 
 We'll put the container up on Dockerhub soon. For now, you can build locally from this repo.
 
-```
+```bash
 docker build -t groundhog -f Dockerfile .
-```
+```bash
 
 ### Running the app
 
 Run this command to kick up a local version of the app
 
-```
+```bash
 docker run -p 5005:5005 --name groundhog_server groundhog
-```
+```bash
 
 You can now hit the API on `localhost`. Run the following command in a separate terminal to see the help docs.
 
-```
+```bash
 curl http://localhost:5005
-```
+```bash
 
 To stop the container by name (if you used the `--name` tag when launching it), do the following:
 
-```
+```bash
 docker stop groundhog_server
-```
+```bash
 
 ### R client
 
 We provide an R client for the service. Given a `data.table` with GPS information and (optionally) bearing, this client will use `groundhog` to enrich that dataset with elevation and slope features.
 
 You can install the package from source
 
-```
+```bash
```

Comment:
You only need to indicate a language on the backticks that BEGIN a code block. Can you please revert the ones that are on the closing codeblocks?

like this:

<img width="88" alt="image" src="https://user-images.githubusercontent.com/7608904/62874426-d2fff300-bce6-11e9-9dbe-e794ab28e406.png">